src/include/platform/NetworkCommissioning.h

```diff
@@ -231,8 +231,8 @@ class WirelessDriver : public Internal::BaseDriver
     virtual uint8_t GetScanNetworkTimeoutSeconds()    = 0;
     virtual uint8_t GetConnectNetworkTimeoutSeconds() = 0;
 
-    virtual Status RemoveNetwork(ByteSpan networkId)                 = 0;
-    virtual Status ReorderNetwork(ByteSpan networkId, uint8_t index) = 0;
+    virtual Status RemoveNetwork(ByteSpan networkId, MutableCharSpan outDebugText, uint8_t * outNetworkIndex) = 0;
+    virtual Status ReorderNetwork(ByteSpan networkId, uint8_t index, MutableCharSpan outDebugText)            = 0;
```

Comment:
The debug text arg should be a `MutableCharSpan &` and the callee should set its length correctly, for all of these functions.